lib/gollum-lib/macro/navigation.rb

```diff
@@ -6,7 +6,7 @@ def render(title = "Navigate in the TOC", toc_root_path = ::File.dirname(@page.p
         if @wiki.pages.size > 0
           list_items = @wiki.pages.map do |page|
             if page.url_path.start_with?(toc_root_path)
-              path_display = full_path ? page.url_path_display  : page.url_path_display.sub(toc_root_path,"").sub(/^\//,'')
+              path_display = full_path ? page.url_path_display  : page.url_path_display.sub(toc_root_path.gsub("-", " "), "").sub(/^\//,'')
```

Comment:
Ah I see, it's another whitespace-substitution issue. We're working on removing all whitespace->dash substitution in `v5.x`, because it's unnecessary and leads to a lot of bugs and oversights like this. @bartkamphorst, would you agree we merge this into `master` as a bugfix though?